plaso/storage/file_interface.py

```diff
@@ -1085,6 +1064,26 @@ def WriteSessionCompletion(self, aborted=False):
     session_completion = self._session.CreateSessionCompletion()
     self._storage_file.WriteSessionCompletion(session_completion)
 
+  def WriteSessionConfiguration(self, knowledge_base):
+    """Writes session configuration information.
+
+    Args:
+      knowledge_base (KnowledgeBase): contains information about the session
+          configuration.
+
+    Raises:
+      IOError: if the storage type does not support writing preprocessing
+          information or when the storage writer is closed.
+      OSError: if the storage type does not support writing preprocessing
+          information or when the storage writer is closed.
+    """
+    self._RaiseIfNotWritable()
+
+    if self._storage_type != definitions.STORAGE_TYPE_SESSION:
+      raise IOError('Session configuration not supported by storage type.')
```

Comment:
Please add the actual storage type to the error message here

Reply:
Same reply as previous comment.